### ISSUE TITLE
CSV to include subject line

### ISSUE THREAD

**Issue by lyndasf** [ePADD/epadd#446]
Could the CSV export from the Processing Module also include the subject lines of message ? I submitted this to Emailchemy as well and never heard back.

Thank you

**Comment by jfarwer** [comment]
Yes sure, that can be done. We will include this in the next release which we plan to do within the next few weeks.

**Comment by lyndasf** [comment]
That's excellent. Looking forward to testing it out.

**Comment by jfarwer** [comment]
This is now implemented in version 10.0.6.
